specs_derive/README.md

```diff
@@ -0,0 +1,107 @@
+# Specs
```

Comment:
I don't think we need a second readme here (or at least, don't copy the specs README and just show a small example of component groups).